dict/prh_bengo4.yml

```
@@ -7,22 +7,25 @@ rules:
     patterns: 弁コム
     prh: サービス名の場合は略さず表記します
   - expected: 弁護士ドットコムLIBRARY
-    patterns: /弁護士ドットコム(Library|library)/
+    patterns: /弁護士ドットコム(\s?[Ll]ibrary)/
```

```suggestion
    patterns: /弁護士ドットコム\s?(L|l)ibrary/
```
こういうことでしょうか？一番外側の()は不要に見えました